packages/x-adapter-platform/src/endpoint-adapters/semantic-queries.endpoint-adapter.ts

```diff
@@ -0,0 +1,18 @@
+import { endpointAdapterFactory } from '@empathyco/x-adapter';
+import { SemanticQueriesResponse, SemanticQueriesRequest } from '@empathyco/x-types';
+import { semanticQueriesRequestMapper } from '../mappers/requests/semantic-queries-request.mapper';
+// eslint-disable-next-line max-len
+import { semanticQueriesResponseMapper } from '../mappers/responses/semantic-queries-response.mapper';
+
+export const semanticQueriesEndpointAdapter = endpointAdapterFactory<
+  SemanticQueriesRequest,
+  SemanticQueriesResponse
+>({
+  endpoint:
+    'https://api.{extraParams.env(.)}empathy.co/semantics-api/search-single/{extraParams.instance}',
```

Comment:
Hi! Shouldn't it be `https://semantics-api.internal.{extraParams.env(.)}empathy.co/search-single/{extraParams.instance}` according to the [Semantics API](https://searchbroker.atlassian.net/wiki/spaces/EAE/pages/4020011011/Semantics+API) docs.

I have tried with this one & updated the tests accordingly. At this point, the semanticQueriesRequestMapper tests were failing due to not passing the instance extraParam, so I am confused (all other tests were passing).

Reply:
Nope, the final URL will be the one I set once the service is deployed to the Empathy API.
[In the component PR](https://github.com/empathyco/x/pull/1213/files#diff-05530d429cfde8d3094b5a04b85cb0c4a5a8f7121fe96548925fbc4ef580f7bf) I set the endpoint you mention provisionally.